cmd/dp-deployer/main.go

```diff
@@ -117,7 +116,33 @@ func main() {
 	case <-ctx.Done():
 		log.Event(ctx, "context done", log.INFO)
 	}
-	wg.Wait()
+
+	log.Event(ctx, fmt.Sprintf("shutdown with timeout: %s", cfg.GracefulShutdownTimeout), log.INFO)
```

Comment:
We like log the message for a log event to be the same every time as this allows us  to search for instances of it in the logs more easily.

The message is fine but at the timeout value as a `log.Data` parameter